tests/test_snapshot_download.py

```diff
@@ -88,3 +90,60 @@ def test_download_model(self):
 
             # folder name contains the revision's commit sha.
             self.assertTrue(self.first_commit_hash in storage_folder)
+
+    def test_download_private_model(self):
```

Comment:
I think this text can be greatly simplified. There's no need to download the repositories, you can use `HfApi.model_info` with and without the token and see if you get an error or if you can get info from the repo.

Reply:
OK sounds good, but then I think the test should be moved to `test_hf_api.py` since we'd no longer be testing a call of `snapshot_download`. I'll push a change so you can see what I mean

Reply:
Discussed offline since there was a misunderstanding from my side